### ISSUE TITLE
Feature request: Disable timeout for XX minutes

### ISSUE THREAD

**Issue by doctapeppa** [tjjfvi/OctoPrint-BetterHeaterTimeout#4]
I sometimes disable this if I'm doing some tuning or messing with the hotend but I'm forgetful and forget to re-enable it. Would it be possible to add a "Disable for XXX Time" so that the timeout re-enable's itself after a period of time?

**Comment by tjjfvi** [comment]
@doctapeppa: Yes, it would. I will not be able to work on this for ~2-3 weeks, but I’ll probably add that after that time.
